.gitignore

```diff
@@ -2,3 +2,5 @@
 .mbed
 projectfiles
 *.py*
+mbed-os/
```

Comment:
Why were these ignores needed?

Reply:
Oh shoot. This one slipped through (had some build issues). Will remove.